.gitignore

```diff
@@ -1,129 +0,0 @@
-# Byte-compiled / optimized / DLL files
-__pycache__/
```

Comment:
Please keep this file as it is and remove the committed files that should be ignored.